exercises/practice/linked-list/.docs/instructions.append.md

```diff
@@ -0,0 +1,6 @@
+# note
```

Comment:
```suggestion
# Notes
```

Reply:
The level-1 header of the append file is discarded by the process that joins the files. It doesn't matter what goes in there.

Reply:
Good to know, but no reason to be sloppy IMHO. :-)